src/Modules/OrchardCore.Commerce/Events/WorkflowShoppingCartEvents.cs

```diff
@@ -21,22 +21,19 @@ public class WorkflowShoppingCartEvents : IShoppingCartEvents
 {
     private readonly IWorkflowManager _workflowManager;
     private readonly IWorkflowTypeStore _workflowTypeStore;
-
-    private readonly JsonSerializerSettings _settings;
+    private readonly IHtmlLocalizer<WorkflowShoppingCartEvents> H;
```

Comment:
in OC we use `protected` or internal for `H` and `S` I don't know if that is the case in this project. just FYI